src/github_auth.js

```diff
@@ -0,0 +1,45 @@
+let Github = require('github');
+let Promise = require('promise');
+let jwt = require('jsonwebtoken');
+
+module.exports = async ({cfg}) => {
+  let github = new Github({
+    promise: Promise,
+  });
+
+  // This object insures that the authentication is delayed until we need it.
+  // Also, the authentication happens not just once in the beginning, but for each request.
+  return {
+    getInstallationGithub: async (inst_id) => {
+      // Authenticating as integration
+      let inteToken = jwt.sign(
+        {iss: cfg.github.credentials.integrationId},
+        cfg.github.credentials.privatePEM,
+        {algorithm: 'RS256', expiresIn: '1m'},
+      );
+      try {
+        github.authenticate({type: 'integration', token: inteToken});
+        debug('Authenticated as integration');
```

Comment:
let's delete this debug, since it will be logged on every successful authentication and doesn't add much